nupic/regions/Spec.py

```diff
@@ -215,7 +215,7 @@ def items2dict(items):
       attributes. The entire items dict will become a dict of dicts (same keys).
       """
       d = {}
-      for k, v in items.items():
+      for k, v in list(items.items()):
```

Comment:
This addition of the unnecessary `list(...)` wrapper by the automatic conversion tool actually highlights the incorrect use of `items()` in our code base in a number of places. Since the body of the loop doesn't alter the `items` dict, the original call should have been `iteritems()`.